src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -40,7 +40,7 @@
   {% set productId = productForm.vars.product_id %}
 
   {{ form_start(productForm, {'attr': {
-    'class': 'form-horizontal product-page row justify-content-md-center', 'novalidate': 'novalidate',
+    'class': 'form-horizontal product-page product-page-v2 row justify-content-md-center', 'novalidate': 'novalidate',
```

Comment:
When we release the V2 page and delete the old page, will we keep this class or remove/replace it?

Reply:
As answered in previous comment I guess it could be cleaned later